src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/mapper/LimitedPartnershipIncorporationMapper.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.companieshouse.limitedpartnershipsapi.mapper;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.factory.Mappers;
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.limitedpartnershipsapi.model.dao.LimitedPartnershipIncorporationDao;
+import uk.gov.companieshouse.limitedpartnershipsapi.model.dto.LimitedPartnershipIncorporationDto;
+
+@Component
+@Mapper(componentModel = "spring")
+public interface LimitedPartnershipIncorporationMapper {
+
+    LimitedPartnershipIncorporationMapper INSTANCE = Mappers.getMapper(LimitedPartnershipIncorporationMapper.class);
+
+    // TODO Map etag
```

Comment:
Will do this when @dlloyd1-ch 's [latest changes](https://github.com/companieshouse/limited-partnerships-api/pull/66) are merged.